### ISSUE TITLE
Release all configurations of GmsCore

### ISSUE THREAD

**Issue by oSumAtrIX** [microg/GmsCore#2257]
TLDR: My question is, which configuration is being released for GmsCore and what is the reason for that? What is the reason for the manual release?

GmsCore comes with three configurations: HMS, Mapbox, and VTM. From what I understand, only one of these configurations is being released because the release page and GitHub release actions only include one APK for GmsCore. 

The release pipeline is not public; there is no workflow, so I assume one configuration is picked, signed, and released manually. Perhaps adding a release workflow so that the release pipeline is transparent may be a good idea.

**Comment by mar-v-in** [comment]
VTM code is no longer maintained and probably doesn't work properly anymore. Mapbox/Maplibre is used in the default version, HMS is used in the -hw version.
